webserver/src/main/java/io/kestra/webserver/controllers/FileController.java

```diff
@@ -0,0 +1,115 @@
+package io.kestra.webserver.controllers;
+
+import io.kestra.core.storages.FileAttributes;
+import io.kestra.core.storages.StorageInterface;
+import io.kestra.core.tenant.TenantService;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.*;
+import io.micronaut.http.multipart.CompletedFileUpload;
+import io.micronaut.http.server.types.files.StreamedFile;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+import io.micronaut.validation.Validated;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import jakarta.inject.Inject;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.*;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.NoSuchFileException;
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@Validated
+@Controller("/api/v1/files")
```

Comment:
```suggestion
@Controller("/api/v1/namespacefiles")
```

Reply:
Are we sure that this should be changed ? To me it does make sens to have `api/v1/files/{namespace}` for this in a rest pov ?